apps/arweave/src/ar_nonce_limiter_server.erl

```diff
@@ -12,6 +12,10 @@
 
 -record(state, {}).
 
+%% @doc The number of steps for which we include step checkpoints in the full session update.
+%% Does not apply to previous session updates.
+-define(SESSION_UPDATE_INCLUDE_STEP_CHECKPOINTS_COUNT, 10).
```

Comment:
What do you thknk about increasing this?

Worst case sizing, assuming N steps of checkpoints returned every second (N * 50 * 32 byets):

10:
Request: 16KB
Per Day: 1.4GB

25:
Request: 40KB
Per Day: 3.5GB

100:
Request: 160KB
Per Day: 14GB

For my miner ~96% of requests yielded 1 step. So these "worse case" numbers are likely pretty inflated. Even a 100 step cap might only add ~600MB of traffic a day. which is a relatively small delta over the current web traffic.

Reply:
Yes, good point, I do not mind trying 100